applications: nrf_desktop: Use high GDFS frequency on nRF54H20 (#20469)
nrfconnect/sdk-nrf: doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

    * Disabled Bluetooth long workqueue (:kconfig:option:`CONFIG_BT_LONG_WQ`).
    * Limited the number of key slots in the PSA Crypto core to 10 (:kconfig:option:`CONFIG_MBEDTLS_PSA_KEY_SLOT_COUNT`).

>   * Disabled :kconfig:option:`CONFIG_CLOCK_CONTROL_NRF2_GLOBAL_HSFLL_REQ_LOW_FREQ` in the IPC radio image configuration for nRF54H20 SoC.

Reviewer: ```suggestion
  * Disabled the :kconfig:option:`CONFIG_CLOCK_CONTROL_NRF2_GLOBAL_HSFLL_REQ_LOW_FREQ` Kconfig option in the IPC radio image configuration for the nRF54H20 SoC.
```

>     The :ref:`nrf_desktop_dvfs` running on the application core selects clock control and enables automatic low GD frequency request (:kconig:option:`CONFIG_CLOCK_CONTROL_NRF2_GLOBAL_HSFLL_REQ_LOW_FREQ`).

Reviewer: ```suggestion
    The :ref:`nrf_desktop_dvfs` running on the application core selects the clock control and enables an automatic low GD frequency request (:kconfig:option:`CONFIG_CLOCK_CONTROL_NRF2_GLOBAL_HSFLL_REQ_LOW_FREQ`).
```

>     The :ref:`nrf_desktop_dvfs` running on the application core selects the clock control and enables an automatic low Global Domain (GD) frequency request (:kconfig:option:`CONFIG_CLOCK_CONTROL_NRF2_GLOBAL_HSFLL_REQ_LOW_FREQ`).

Reviewer 2: That is not correct, or complete picture. To make the GD frequency low all local domains has to setup this frequency. The CONFIG_CLOCK_CONTROL_NRF2_GLOBAL_HSFLL_REQ_LOW_FREQ is enabled because the nrf2 clock control is enabled. I don't know what enables it in application core. In radio core the nrf2 clock control is enabled by MPSL.

Author: I emphasized what enables it on the app core here:
```
The :ref:`nrf_desktop_dvfs` running on the application core selects the clock control and enables an automatic low Global Domain (GD) frequency request (:kconfig:option:`CONFIG_CLOCK_CONTROL_NRF2_GLOBAL_HSFLL_REQ_LOW_FREQ`).
```

Author: Introduced a bit more details

>     Bluetooth stack with SoftDevice Link Layer requires high frequency in GD to make L2 cache and MRAM run fast to avoid code execution delays.

Reviewer: ```suggestion
    Bluetooth stack with the SoftDevice Link Layer requires high frequency in GD to make the L2 cache and MRAM run fast to avoid code execution delays.
```
What is GD? Should we open it?

Author: Global Domain. We can open it I think

Reviewer: But you did not?

    * Disabled Bluetooth long workqueue (:kconfig:option:`CONFIG_BT_LONG_WQ`).
    * Limited the number of key slots in the PSA Crypto core to 10 (:kconfig:option:`CONFIG_MBEDTLS_PSA_KEY_SLOT_COUNT`).

>   * Disabled the :kconfig:option:`CONFIG_CLOCK_CONTROL_NRF2_GLOBAL_HSFLL_REQ_LOW_FREQ` Kconfig option in the IPC radio image configuration for the nRF54H20 SoC (the option was enabled by default, because clock control is enabled by the MPSL - Multiprotocol Service Layer).

Reviewer: ```suggestion
  * Disabled the :kconfig:option:`CONFIG_CLOCK_CONTROL_NRF2_GLOBAL_HSFLL_REQ_LOW_FREQ` Kconfig option in the IPC radio image configuration for the nRF54H20 SoC.
    The option was enabled by default, because the clock control is enabled by the Multiprotocol Service Layer (MPSL).
```
